test/e2e/framework/autoscaling/autoscaling_utils.go

```diff
@@ -372,20 +372,23 @@ func (rc *ResourceConsumer) GetReplicas() int {
 		if replicationController == nil {
 			framework.Failf(rcIsNil)
 		}
+		//lint:ignore SA5011 nil check happens above, can't be nil.
```

Comment:
what about checking `replicationController.Status` for `nil`?

/close

Please re-open if you plan to work on this PR.